### ISSUE TITLE
A bad advertisement

### ISSUE THREAD

**Issue by White-Rabbit-Scientific** [gskinnerTeam/flutter-folio#83]
This project is a bad advertisement for Flutter. It fails to run on multiple platforms, as evidenced in the comments.

**Comment by sk92129** [comment]
> This project is a bad advertisement for Flutter. It fails to run on multiple platforms, as evidenced in the comments.

Agreed -- this repo project is referenced from the https://docs.flutter.dev/ui/layout/responsive/building-adaptive-apps flutter site so they should keep this project up to date and buildable.

**Comment by sparklton** [comment]
Just tried the Windows demo, it simply doesn't work as the keyboard inputs for login are not functioning. And it was reported quite some time ago.. is this project abandoned?
